modules/services/imapnotify.nix

```diff
@@ -0,0 +1,94 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.imapnotify;
+
+  imapnotifyAccounts =
+    filter (a: a.imapnotify.enable) (attrValues config.accounts.email.accounts);
+
+  genAccountUnit = account: {
+    name = "imapnotify@${account.address}";
+    value = {
+      Unit = {
+        Description = "Execute scripts on IMAP mailbox changes (new/deleted/updated messages) using IDLE for %i";
+        PartOf = [ "network-online.target" ];
```

Comment:
There is no `network-online.target` for user services. As far as I know one would have to set up some alternative target that attempts to detect when there is connectivity or to rely on the service gracefully handling the no network situation.

Reply:
You're right, I don't know why I threw that in there; I had it lying in an old config. As far as I'm aware it shouldn't misbehave if there's no internet access. I'll check later